app/src/main/resources/explorer-backend-openapi.json

```diff
@@ -6219,13 +6219,12 @@
               "schema": {
                 "type": "array",
                 "items": {
-                  "type": "string",
-                  "format": "32-byte-hash"
+                  "type": "string"
                 }
               },
               "example": [
-                "2d11fd6c12435ffb07aaed4d190a505b621b927a5f6e51b61ce0ebe186397bdd",
-                "bd165d20bd063c7a023d22232a1e75bf46e904067f92b49323fe89fa0fd586bf"
+                "ALPH",
+                "USDT"
```

Comment:
Maybe we should provide the complete list? Or make it an enum?

Otherwise, people might not know what to provide